packages/data-service/src/data-service.ts

```diff
@@ -0,0 +1,709 @@
+import { EventEmitter } from 'events';
+import {
+  AggregateOptions,
+  AggregationCursor,
+  BulkWriteOptions,
+  ClientSession,
+  Collection,
+  CollectionInfo,
+  ConnectionOptions,
+  CountDocumentsOptions,
+  CreateCollectionOptions,
+  CreateIndexesOptions,
+  DeleteOptions,
+  DeleteResult,
+  Document,
+  EstimatedDocumentCountOptions,
+  ExplainOptions,
+  Filter,
+  FindCursor,
+  FindOneAndReplaceOptions,
+  FindOneAndUpdateOptions,
+  FindOptions,
+  IndexSpecification,
+  InsertManyResult,
+  InsertOneOptions,
+  InsertOneResult,
+  ServerClosedEvent,
+  ServerDescriptionChangedEvent,
+  ServerOpeningEvent,
+  TopologyClosedEvent,
+  TopologyDescription,
+  TopologyDescriptionChangedEvent,
+  TopologyOpeningEvent,
+  UpdateFilter,
+  UpdateOptions,
+  UpdateResult,
+} from 'mongodb';
+import NativeClient, {
+  ConnectionModel,
+  NativeClient as NativeClientType,
+} from './native-client';
+import { Callback, Instance } from './types';
+
+class DataService extends EventEmitter {
+  /**
+   * Stores the most recent topology description from the server's SDAM events:
+   * https://github.com/mongodb/specifications/blob/master/source/server-discovery-and-monitoring/server-discovery-and-monitoring-monitoring.rst#events
+   */
+  lastSeenTopology: TopologyDescription | null = null;
+
+  client: NativeClientType;
+
+  constructor(model: ConnectionModel) {
+    super();
+
+    /* eslint-disable */
```

Comment:
Yeah this can now be removed due to the changed eslint config. However, the lines will hit all issues of the no-unsafe stuff, so disabling for next line just doesn't cut it here and looks super ugly, too :D

Reply:
Ahhh, gotcha, yeah, makes sense in that case 😄